manim/mobject/geometry.py

```diff
@@ -1569,6 +1573,8 @@ def coordinate_label(
             The number of dimensions of the vector.
         color
             The color of the label.
+        show_bg_rec
+            Display background rectangle for a number.
 
         Examples
```

Comment:
The example for the method isn't showing up, as shown [here](https://manimce--1988.org.readthedocs.build/en/1988/reference/manim.mobject.geometry.Vector.html). Could you fix that? The indentation of the manim code block isn't quite right.

Reply:
I didn't quite get this one. I saw the page and yes the example is missing. But upon checking the indentation in the function docstring and committing, I couldn't pinpoint the exact issue that could be causing the missing example. I tried crosschecking with the other examples present in the script as well. Will try investigating a bit more.

Reply:
Ah sorry, I misjudged,  the problem was at the `.. manim` keyword at the start of the example, it needs a double colon

Reply:
Okay, I tried that as well. Then thought maybe a missing "example" word in the name could be an issue but even now, on refreshing, the example is not getting displayed.

Reply:
Oh yes, found it. There was some build issue too due to wrong variable name in the example. Fixed both. Am able to see the example in the link now.